broker/src/main/java/io/zeebe/broker/system/configuration/BrokerCfg.java

```diff
@@ -23,6 +23,8 @@
   private EmbeddedGatewayCfg gateway = new EmbeddedGatewayCfg();
   private BackpressureCfg backpressure = new BackpressureCfg();
 
+  private String stepTimeout = "5m";
```

Comment:
Should this be exposed to users? If yes, we should also add it to the config file for documentation.

Reply:
actually I forgot to do this yes. Thanks.